templates/policy/route-map/node.tag/rule/node.tag/set/extcommunity-rt/node.def

```diff
@@ -2,7 +2,8 @@ type: txt
 help: Set route target value
 val_help: ASN:nn_or_IP_address:nn VPN extended community
 
-syntax:expression: pattern $VAR(@) "\d+:\d+(\.\d+\.\d+\.\d+):\d+" ; "Should be in form: ASN:nn_or_IP_address:nn where ASN is autonomous system number" 
+syntax:expression: exec "${vyos_libexec_dir}/validate-value.py --regex \'^((?:[0-9]{1,3}\\.){3}[0-9]{1,3}|\\d+):\\d+$\'  --value \'$VAR(@)\'"; "Should be in form: ASN:NN or IPADDR:NN where ASN is autonomous system number"
```

Comment:
Do we have such validator?
```
vyos@r4-1.3# set policy route-map FOO rule 10 set extcommunity-rt 10:20

  sh: /usr/libexec/vyos/validate-value.py: No such file or directory
  
  Should be in form: ASN:NN or IPADDR:NN where ASN is autonomous system number
  Value validation failed
  Set failed

```

Reply:
Good catch - Validators are named without extension on VyOS 1.3. This PR has been updated.